tutorials/outflux.md

```diff
@@ -52,7 +54,8 @@ $ influx -import -path=outflux_taxit.txt -database=outflux_tutorial
 2019/03/27 11:39:11 Failed 0 inserts
 ```
 
-The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls.
+The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. 
+All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls.
```

Comment:
Nit:  Oxford comma for consistency:  "location, rating, and vendor"   "fare, mta_tax, tip, and tolls."